ansible/templates/etc/default/pyslackersweb.j2

```diff
@@ -7,4 +7,5 @@ SECRET_KEY={{ secret_key }}
 EMAIL_USER={{ sendgrid.username }}
 EMAIL_PASSWORD={{ sendgrid.password }}
 SLACK_JOIN_CHANNELS={{ slack.join_channels }}
-SLACK_OAUTH_TOKEN={{ slack.oauth_token }}
\ No newline at end of file
+SLACK_OAUTH_TOKEN={{ slack.oauth_token }}
+APP_GIT_REVISION={{ clone.after }}
```

Comment:
I _think_ for this to work right you will want to pass in variables to the [task that places this file](https://github.com/pyslackers/website/blob/e4048d3df3102acf7525e4fe18743f7146af2ba6/ansible/playbook.yml#L91). It's ideal for two reasons:

1. Explicit
2. The templates should be fairly dumb and not know about task results, it just needs to be told what to put here.

Reply:
So to make it work .. I need to create a variable in tasks that places this file like
vars : 
      "app_revision" : {{clone.after}}

and then use this variable in this file like 
APP_GIT_REVISION={{app_revision}}
??